stablehlo/dialect/StablehloOps.td

```diff
@@ -1239,10 +1240,16 @@ def StableHLO_SendOp : StableHLO_Op<"send", [
     Variadic<HLO_TensorOrPerAxisQuantizedTensor>:$inputs, /*send_i1*/
     HLO_Token:$token, /*send_i2*/
     StableHLO_ChannelHandle:$channel_handle, /*send_i3_i4*/
-    DefaultValuedOptionalAttr<BoolAttr, "false">:$is_host_transfer /*send_i5*/
+    DefaultValuedOptionalAttr<BoolAttr, "false">:$is_host_transfer, /*send_i5*/
+    OptionalAttr<I64ElementsAttr>:$source_target_pairs /*send_i6*/
   );
 
   let results = (outs HLO_Token);
+  let builders = [
+    OpBuilder<(ins
+      "::mlir::Type":$result_type, "::mlir::Value":$operand,
+      "::mlir::DenseIntElementsAttr":$source_target_pairs,
+      "::mlir::stablehlo::ChannelHandleAttr":$channel_handle)>];
```

Comment:
I'm seeing the below error when I bump the submodule in IREE. I think you missed the implementation for the builders. It also happens in the other op. My local cmake build is happy if I just drop the builders: https://github.com/iree-org/stablehlo/commit/28d5a80714a35078aee1a1672f8579cf66339e4a

```
error: undefined symbol: mlir::stablehlo::RecvOp::build(mlir::OpBuilder&, mlir::OperationState&, mlir::Type, mlir::Value, mlir::DenseIntElementsAttr, mlir::stablehlo::ChannelHandleAttr)
```